### PR TITLE
Dummy session route

### PR DESCRIPTION
**Pull request by m4v15**
Get rid of the route from the auth router that we were using for tests and put it in the test file itself.

### CODE REVIEW

File: tests/middleware/sessionMiddleWare.test.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+const { addUserWithHashedPassword } = require('../helpers/index.js')
+const { makeLoggedInToken } = require('../../src/controllers/session.js')
+
+const validateJWT = require('../../src/middleware/validateJWT.js')
+const validateUser = require('../../src/middleware/validateUser.js')
+const checkRole = require('../../src/middleware/rolePermission.js')
+const boomErrors = require('../../src/middleware/boomErrorHandler.js')
+
+// dummy secure route to test on
+server.get('/apps',
```
Comment (mattlub) [inline tests/middleware/sessionMiddleWare.test.js]: change to `/test` or simliar?

File: tests/middleware/sessionMiddleWare.test.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  validateJWT(),
+  validateUser,
+  checkRole({ minRole: roles.SUPER }),
+  (req, res, next) => res.send('RESULTS')
+)
+server.use(boomErrors)
```
Comment (mattlub) [inline tests/middleware/sessionMiddleWare.test.js]: put this inside the previous brackets? So like:
```js
server.get('/apps',
...
boomErrorHandler
)
```

**Review by mattlub — CHANGES_REQUESTED** [review]

**Comment by m4v15** [comment]
@mattlub addressed

**Review by mattlub — APPROVED** [review]

**Review by des-des — APPROVED** [review]

**Comment by des-des** [comment]
@m4v15 happy for you to merge

**Pull request merged**